src/SourceCode.Clay/SemanticVersionIncompatabilities.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SourceCode.Clay
+{
+    /// <summary>
+    /// Represents the different levels of <see cref="SemanticVersion"/> compatability.
+    /// </summary>
+    [Flags]
+    public enum SemanticVersionIncompatabilities
```

Comment:
Is it not possible to name this as a positive? Incompatibilities might often end up being used in a double-negative context: `if (!(foo & Incompat > 0))...`